distributed/variable.py

```diff
@@ -130,6 +130,15 @@ class Variable:
 
        This object is experimental and has known issues in Python 2
 
+    Parameters
+    ----------
+    name: string (optional)
+        Name used by other clients and the scheduler to identify the variable.
+        If not given, a random name will be generated.
+    client: Client (optional)
+        Client used for communication with the scheduler. Defaults to the
+        value of ``_get_global_client()``.
```

Comment:
We can apply the same change here as well. Though let's keep the discussion in the first thread for simplicity 🙂